### PR TITLE
Introduce roundcube_working_dir variable.

### PR DESCRIPTION
**Pull request by neidiom**
My Control Panel by default sets the user home dir to a different location than `roundcube_user_home` so every time I run this role it sets it to `roundcube_user_home` and then the Control Panel changes it back. This PR fixes this.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ roundcube_support_url: https://www.systemli.org/kontakt.html
 roundcube_user: roundcube
 roundcube_group: roundcube
 roundcube_user_shell: /bin/false
+roundcube_user_home_directory: "/var/www/{{ roundcube_domain }}"
```
Comment (doobry-systemli) [inline defaults/main.yml:11]: I think this approach is a bit confusing for people who don't face the issues you face. Maybe instead we could introduce a `roundcube_home` variable that defaults to `"{{ roundcube_user_home }}` and use that variable everywhere except for the task that creates the user? What do you think @t2d?

**Review by doobry-systemli — CHANGES_REQUESTED** [review]

**Comment by t2d** [comment]
The naming of the variables is definitely confusing. But I also don't get the problem. 
@neidiom you should adjust `roundcube_user_home` to the one provided by your control panel. Then you should set `roundcube_path` to the directory where you want to install roundcube. Shouldn't that be enough?

**Comment by neidiom** [comment]
@t2d I tried that before and it does not work. The role breaks. I am using ISPConfig which uses sym links for the web folder as following

`Website path - /var/www/clients/client[client_id]/web[website_id]`
`Website symlinks - /var/www/[website_domain]/:/var/www/clients/client[client_id]/[website_domain]/`

The `Website path` has special permissions via `chattr` so no new files can be created outside the web directory.

**Comment by neidiom** [comment]
But I agree that the variable naming is confusing. Maybe instead of `roundcube_user_home_directory` I change it to `system_user_home_directory` @t2d ?

**Comment by neidiom** [comment]
So in my example the links are set as this

````
lrwxrwxrwx 1 root root 31 Apr 12 06:23 /var/www/xxx.mydomain.com -> /var/www/clients/client1/web17/
````

````
# ls -al /var/www/xxx.mydomain.com/web/ | grep www
lrwxrwxrwx  1 web17     client1        66 Apr 17 18:06 www -> /var/www/clients/client1/web17/web/roundcubemail-1.6.1/public_html
````

**Comment by neidiom** [comment]
Also the Control Panel has `open_basedir` restrictions set so the files need to be inside the web folder.

```
grep open /etc/php/8.1/fpm/pool.d/web17.conf 
php_admin_value[open_basedir] = /var/www/clients/client1/web17/web:/var/www/clients/client1/web17/private:/var/www/clients/client1/web17/tmp
```

**Comment by neidiom** [comment]
@t2d I changed the name of the variable so that it id less confusing.

**Comment by t2d** [comment]
Thanks for you work on this. However, due to ansible convention every variable must start with `roundcube_`. Please go the other way around. `roundcube_user_home` should define the home of the roundcube user. But you can add another variable e.g. `roundcube_working_dir` with default `roundcube_working_dir = {{ roundcube_user_home }}` and then override these variables in your case. Do you understand what I mean?

**Comment by neidiom** [comment]
@t2d Please review the changes.

**Comment by t2d** [comment]
Great, exactly what I imagined.

**Pull request merged**